src/Inode.php

```diff
@@ -237,9 +169,19 @@ public function chmod(int $permissions): vfsStreamContent
     }
 
     /**
-     * returns permissions
+     * @deprecated  use permissions() instead
```

Comment:
Maybe it's just me, but I like the get/set prefixes because it makes it very obvious what the method does.

I might just be lazy and not like reading a method's description to find out what it's doing. An example:
```php
$file->lastAttributeModified($filectime); // takes parameter = setter
$file->filectime(); // no parameter = getter

// but then...

$file->resourceId($resource); // takes parameter, but is a getter not a setter
```

Reply:
You are right, `resourceId($resource)` is not a good method name for what it does. Should be something like `resourceIdOf($resource)` or `resourceIdFor($resource)` I think.